kernelci/db/kernelci_api.py

```diff
@@ -57,6 +58,16 @@ def _put(self, path, data=None):
         resp.raise_for_status()
         return resp
 
+    def create_user(self, username, password, is_admin, verbose=False):
+        """Create new user"""
+        path = '/'.join(['user', username])
+        params = {'is_admin': is_admin}
+        data = {'password': password}
+        resp = self._post(path, data=json.dumps(data), params=params)
+        if verbose:
+            print(resp.json())
```

Comment:
I think the method should return the response and then kci_data can print something with verbose flag enabled instead.

Reply:
Done.